tests/resources/sample_input_census_spec_england_questionnaire.csv

```diff
@@ -0,0 +1,7 @@
+ARID,ESTAB_ARID,UPRN,ADDRESS_TYPE,ESTAB_TYPE,ADDRESS_LEVEL,ABP_CODE,ORGANISATION_NAME,ADDRESS_LINE1,ADDRESS_LINE2,ADDRESS_LINE3,TOWN_NAME,POSTCODE,LATITUDE,LONGITUDE,OA,LSOA,MSOA,LAD,REGION,HTC_WILLINGNESS,HTC_DIGITAL,FIELDCOORDINATOR_ID,FIELDOFFICER_ID,TREATMENT_CODE,CE_EXPECTED_CAPACITY
+DDR190314000000195675,DDR190314000000113740,,HH,Household,U,RD06,,Flat 56 Francombe House,Commercial Road,,Windleybury,XX1 0XX,51.4463421,-2.5924477,E00073438,E01014540,E02003043,E06000023,E12000009,1,5,1,2,HH_QF2R1E,3
+DDR190314000000195675,DDR190314000000113741,,HH,Household,U,RD06,,Flat 566 Francombe House,Commercial Road,,Windleybury,XX1 0XX,51.4463421,-2.5924477,E00073438,E01014540,E02003043,E06000023,E12000009,1,5,1,2,HH_QF2R2E,3
+DDR190314000000195675,DDR190314000000113742,,HH,Household,U,RD06,,Flat 561 Francombe House,Commercial Road,,Windleybury,XX1 0XX,51.4463421,-2.5924477,E00073438,E01014540,E02003043,E06000023,E12000009,1,5,1,2,HH_QF2R3AE,3
+DDR190314000000195675,DDR190314000000113743,,HH,Household,U,RD06,,Flat 562 Francombe House,Commercial Road,,Windleybury,XX1 0XX,51.4463421,-2.5924477,E00073438,E01014540,E02003043,E06000023,E12000009,1,5,1,2,HH_QF3R1E,3
+DDR190314000000195675,DDR190314000000113744,,HH,Household,U,RD06,,Flat 563 Francombe House,Commercial Road,,Windleybury,XX1 0XX,51.4463421,-2.5924477,E00073438,E01014540,E02003043,E06000023,E12000009,1,5,1,2,HH_QF3R2E,3
+DDR190314000000195675,DDR190314000000113745,,HH,Household,U,RD06,,Flat 564 Francombe House,Commercial Road,,Windleybury,XX1 0XX,51.4463421,-2.5924477,E00073438,E01014540,E02003043,E06000023,E12000009,1,5,1,2,HH_QF3R3AE,3
```

Comment:
What is the purpose of this file? It fails validation since the ARIDs aren't unique.

Reply:
I used this for testing locally. I will delete it.